src/btc/mod.rs

```diff
@@ -158,16 +160,29 @@ impl_serde_str! { PublicKey }
 impl_serde_str! { Address }
 impl_serde_str! { InputSignature }
 
-/// Generates public and secret keys for Bitcoin node
-/// using given random number generator.
+/// Generate Bitcoin keypair using the given random number generator.
 pub fn gen_keypair_with_rng<R: Rng>(rng: &mut R, network: Network) -> (PublicKey, PrivateKey) {
     let (pk, sk) = secp_gen_keypair_with_rng(rng, network);
     (PublicKey(pk), PrivateKey(sk))
 }
 
 /// Same as [`gen_keypair_with_rng`](fn.gen_keypair_with_rng.html)
-/// but it uses default random number generator.
+/// but it uses a default random number generator.
 pub fn gen_keypair(network: Network) -> (PublicKey, PrivateKey) {
     let (pk, sk) = secp_gen_keypair(network);
     (PublicKey(pk), PrivateKey(sk))
 }
+
+/// Convert Bitcoin sha256d type to the Exonum hash type.
+pub fn sha256d_to_exonum_hash(hash: sha256d::Hash) -> crypto::Hash {
+    let mut bytes = [0_u8; 32];
+    bytes.copy_from_slice(&hash[..]);
+    bytes.reverse();
+    crypto::Hash::new(bytes)
+}
+
+/// Convert Exonum hash type to the Bitcoin sha256d.
```

Comment:
I think we should add a note why we simply copy the hash, without the additional hashing.